colossus-docs/src/main/paradox/streaming.md

```diff
@@ -136,3 +136,11 @@ body of the request/response is a stream of message chunks.
 
 @@snip [StreamingHttpExample.scala]($examples$/StreamingHttpExample.scala) { #streaming_http }
 
+## Enabling compression for the streaming services

```

Comment:
Shorter title would be better since it shows up in the side bar `## Compression`